helm/values.yaml

```diff
@@ -21,9 +21,16 @@ service:
   port: 443
 
 # configure access to the notary server
-notaries:
-# public notary from dockerhub
-- name: dockerhub
+validators:
+- name: allow
+  type: static
+  approve: true
+- name: deny
+  type: static
+  approve: false
+- name: default
+# public notary form dockerhub
+  type: notaryv1
   host: notary.docker.io
   pub_root_keys:
```

Comment:
as I see it, there is now `pub_root_keys`, `keys` (for cosign) and `root_keys` (in the comment) for setting keys. Do we need all of those or could we collapse this into 1 key value like `pubkeys` or something?

Reply:
we can do that. for notary things might get confusing as there are so many keys flying around and pub_root_keys makes things more clear. I don't really have any stakes in this and can settle with just calling them `keys` or `pub_keys`.

Reply:
Ok. Let's see. I would prefer to call them `pub` something to make sure people are aware that it is only public keys, otherwise we might have people adding a private key.
To the confusion about keys: cosign currently only has one type of keys and those are not "root" as there is no hierarchy. So that would mean, we need to add something like `pubkeys` or `pub_keys` (not sure which one is advised. Now for notary v1, there is different types of keys, but in connaisseur you can only specify the public root key and that seems unlikely to change, specifically as I do not see nv1 changing much any time soon or ever again. Consequently, it is only a matter of whether someone would think it might make sense to add a delegation key there (to my knowledge the only other keys in nv1). At that point, that someone is attempting a more complicated use case and would have had to take the extra steps to create a delegation key to begin with. I presume those uses will take the time to read our hopefully understandable docs.
As such, I would see no problem in calling it e.g. `pub_keys` in both cases to keep things simple and consistent. Would you see a case where this might become conflicting?